front_end/views/home.html

```diff
@@ -0,0 +1,147 @@
+      <label for="position">Position</label>
+      <div class="form-group row" id="position">
+          <div class="col-xs-2">
+            <label for="x">x</label>
+            <input class="form-control" id="x" type="text">
+          </div>
+          <div class="col-xs-2">
+            <label for="y">y</label>
+            <input class="form-control" id="y" type="text">
+          </div>
+          <div class="col-xs-2">
+            <label for="z">z</label>
+            <input class="form-control" id="z" type="text">
+          </div>
+      </div>
+
+      <div class="form-group">
+          <label for="colour">Colour</label>
+          <input type="text" class="form-control" id="color">
+      </div>
+
+      <label for="material">Material</label>
+      <div class="form-group" id="material">
+          <select class="form-control" id="material">
+            <option>Plastic</option>
+            <option>Metal</option>
+            <option>Glass</option>
+          </select>
+        </div>
+
+        <br>
+        <h3>Choose environment specification</h3>
+
+        <div class="form-group">
+            <label for="filename">Filename</label>
+            <input type="text" class="form-control" id="filename">
+        </div>
+
+        <label for="scene">Scene Size</label>
+        <div class="form-group row" id="scene">
+            <div class="col-xs-2">
+              <label for="x">x</label>
+              <input class="form-control" id="x" type="text">
+            </div>
+            <div class="col-xs-2">
+              <label for="y">y</label>
+              <input class="form-control" id="y" type="text">
+            </div>
+        </div>
+
+        <div class="form-group">
+            <label for="background">Background</label>
+            <input type="text" class="form-control" id="background">
+        </div>
+
+
+        <h4>Choose light specification</h4>
+
+        <label for="position">Position</label>
+        <div class="form-group row" id="position">
+            <div class="col-xs-2">
+              <label for="x">x</label>
+              <input class="form-control" id="x" type="text">
+            </div>
+            <div class="col-xs-2">
+              <label for="y">y</label>
+              <input class="form-control" id="y" type="text">
+            </div>
+            <div class="col-xs-2">
+              <label for="z">z</label>
+              <input class="form-control" id="z" type="text">
+            </div>
+        </div>
+  
+        <div class="form-group">
+            <label for="colour">Colour</label>
+            <input type="text" class="form-control" id="color">
+        </div>
+
+        <div class="form-group">
+            <label for="intensity">Intensity</label>
+            <input type="text" class="form-control" id="intensity">
+        </div>
+
+        <h4>Choose camera specification</h4>
+
+        <label for="position">Position</label>
+        <div class="form-group row" id="position">
```

Comment:
same with the ids here... basically all of them so that we can access them directly
Stuff like `camera_position` would be better

Reply:
Nice one, closing comment.